prober/http.go

```diff
@@ -297,6 +351,11 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 			Help: "Indicates if probe failed due to regex",
 		})
 
+		probeFailedDueToCel = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
I'm thinking this should be added conditionally, if the configuration asks for validation using CEL expressions.

Reply:
I was following the same logic that is applied to regex checks, following your logic that should also be conditionally added, no?